autoload/ale/fixers/buf_format.vim

```diff
@@ -0,0 +1,7 @@
+" Description: Run buf format.
+
+function! ale#fixers#buf_format#Fix(buffer) abort
+    return {
+    \   'command': 'buf format %s',
```

Comment:
I don't actually know if this works - a lot of the fixers in ale rely on the command taking stdin and stdout, see i.e. https://github.com/dense-analysis/ale/blob/master/autoload/ale/fixers/cmakeformat.vim